examples/adv_diff/ex7/LevelSetInitialCondition.cpp

```diff
@@ -22,10 +22,12 @@
 /////////////////////////////// PUBLIC ///////////////////////////////////////
 
 LevelSetInitialCondition::LevelSetInitialCondition(const std::string& object_name,
+                                                   const Pointer<CartesianGridGeometry<NDIM> > grid_geom,
                                                    const double radius,
                                                    const IBTK::VectorNd& origin,
                                                    const bool fluid_is_interior_to_cylinder)
     : d_object_name(object_name),
+      d_grid_geom(grid_geom),
```

Comment:
Same here: this should be after `d_origin`

Reply:
Are we looking at the same header? In `examples//adv_diff/ex7/LevelSetInitialCondition.h` I see

```cpp

    /*!
     * Name of this object.
     */
    std::string d_object_name;

    /*!
     * Radius of cylinder.
     */
    double d_radius;

    /*!
     * Origin of cylinder.
     */
    IBTK::VectorNd d_origin;

    /*!
     * The Cartesian grid geometry object provides the extents of the
     * computational domain.
     */
    SAMRAI::tbox::Pointer<SAMRAI::geom::CartesianGridGeometry<NDIM> > d_grid_geom;

    /*!
     * Boolean to identify whether fluid is considered inside the cylinder or outside.
     * Default is set false to denote the annulus case.
     */
    bool d_fluid_is_interior_to_cylinder = false;
};
```

Reply:
You are right. Now, I incorporated your suggestions and also added a change log. Please take a look